third_party/terraform/data_sources/data_source_google_kms_key_ring.go

```diff
@@ -0,0 +1,55 @@
+package google
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func dataSourceGoogleKmsKeyRing() *schema.Resource {
```

Comment:
the resource-related tests/func for this do not have `google` in the name: don't know if you have a standard, but I just went with how other data soures are named